### PR TITLE
SVG elements' className is of the SVGAnimatedString type instead of string (close #354)

### LINKED ISSUE #354: Javascript error on wikia.com
https://testcafe-hhhm.devexpress.com/history?timestamp=1452625422874&siteUrl=http://www.wikia.com/Wikia - reproduces in all last runs

### PR DESCRIPTION
**Pull request by churkin**
\cc @LavrovArtem @miherlosev

### CODE REVIEW

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 1]
```diff
@@ -262,9 +262,17 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                     var elementCount = 0;
 
                     for (var i = 0; i < collection.length; i++) {
-                        if (collection[i].className &&
-                            collection[i].className.indexOf(SHADOW_UI_CLASSNAME.postfix) !== -1)
-                            elementCount++;
+                        var className = collection[i].className;
+
+                        if (className) {
+                            // NOTE: SVG elements\' className is of the SVGAnimatedString type instead
```
Comment (LavrovArtem) [inline src/client/sandbox/code-instrumentation/properties/index.js]: Extra slash ```elements\'```

**Comment by churkin** [comment]
FPR

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 2]
```diff
@@ -262,9 +262,17 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                     var elementCount = 0;
 
                     for (var i = 0; i < collection.length; i++) {
-                        if (collection[i].className &&
-                            collection[i].className.indexOf(SHADOW_UI_CLASSNAME.postfix) !== -1)
-                            elementCount++;
+                        var className = collection[i].className;
+
+                        if (className) {
+                            // NOTE: SVG elements' className is of the SVGAnimatedString type instead
```
Comment (miherlosev) [inline src/client/sandbox/code-instrumentation/properties/index.js]: lgtm.
@VasilyStrelyaev check comment and merge it

**Comment by LavrovArtem** [comment]
lgtm

File: src/client/sandbox/code-instrumentation/properties/index.js (hunk 2 above)
Comment (VasilyStrelyaev) [inline src/client/sandbox/code-instrumentation/properties/index.js]: already done in private

**Pull request merged**